api/app/controllers/spree/api/v2/storefront/stores_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class StoresController < ::Spree::Api::V2::BaseController
+          def show
+            render_serialized_payload { serialize_resource(resource) }
+          end
+
+          private
+
+          def resource
+            Spree::Store.find_by(params[:id])
+          end
+
```

Comment:
```suggestion
          def scope
            Spree::Store.accessible_by(current_ability, :show)
          end

          def resource
            scope.find_by(code: params[:id])
          end
```

Probably it's better to use code rather than expose the ID

Reply:
I agree. I have updated the code to reflect what you have suggested. Thanks!